cmd/dashboards/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/grafana/grafana-foundation-sdk/go/dashboard"
+
+	"github.com/grafana/cloudcost-exporter/cloudcost-exporter-dashboards/dashboards"
+)
+
+func main() {
+	mode := flag.String("mode", "json", "output mode. Can be json or file")
+	outputDir := flag.String("output-dir", "./cloudcost-exporter-dashboards/grafana", "output directory")
+	flag.Parse()
+	dashes := dashboards.BuildDashboards()
+
+	err := run(dashes, mode, outputDir)
+	if err != nil {
+		log.Fatalf("error generating dashboards: %s", err.Error())
+	}
+}
+
+func run(dashes []*dashboard.DashboardBuilder, mode *string, outputDir *string) error {
+	for _, dash := range dashes {
+		build, err := dash.Build()
+		if err != nil {
+			return err
+		}
+		output, err := json.MarshalIndent(build, "", "  ")
+		if err != nil {
+			return err
+		}
+		if *mode == "json" {
+			fmt.Println(string(output))
+			continue
+		}
```

Comment:
I'm sure there's a way here to pass in something that implements the writer interface and we call w.Write, will revisit this if it's pressing.